.gitignore

```diff
@@ -2,13 +2,13 @@
 /web/css/
 /web/js/
 /web/uploads/
+/web/fonts/
 /app/bootstrap.php.cache
 /app/cache/*
 /app/config/parameters.yml
 /app/logs/*
 !app/cache/.gitkeep
 !app/logs/.gitkeep
-/app/data/*
```

Comment:
Are u sure u wanna remove this folder ?? IDC but others might loose their db ...